packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+export type CardHeaderProps = {
+  headingWithDefaultServiceFtlId: string;
```

Comment:
If someone's passing in `headingWithCustomServiceFtlId` do they really also need to pass in `headingWithDefaultServiceFtlId`? This might could use a type improvement as well similar to the other callout, I think it's OK to just leave a TODO here with a note about a type improvement or file a 1 pointer to address any of them.

Reply:
Added a TODO comment